apps/fulfillment/mock/index.js

```diff
@@ -0,0 +1,8 @@
+require('dotenv').config();
+const { createMockServer } = require('./server');
+
+const port = process.env.MOCK_PORT_PA || 3001;
```

Comment:
It's not a PA (PickingApp) but Fulfillment so let's rename it:
```suggestion
const port = process.env.ORYX_FULFILLMENT_MOCK_PORT || 3001;
```